Xamarin.Forms.Platform.Android/CollectionView/EmptySource.cs

```diff
@@ -7,5 +7,25 @@ internal class EmptySource : IItemsViewSource
 		public int Count => 0;
 
 		public object this[int index] => throw new IndexOutOfRangeException("IItemsViewSource is empty");
+
+		public void Dispose()
+		{
+			Dispose(true);
+		}
+
+		bool _disposed;
+
+		protected virtual void Dispose(bool disposing)
```

Comment:
We can just mark this as `sealed` and then we don't have to implement the `protected virtual void Dispose(bool disposing)` pattern.